app/javascript/components/patient/laboratory/FirstPositiveLaboratory.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Form } from 'react-bootstrap';
+import moment from 'moment';
+
+import confirmDialog from '../../util/ConfirmDialog';
+import LaboratoryModal from './LaboratoryModal';
+
+class FirstPositiveLaboratory extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+    };
+  }
+
+  handleDelete = async () => {
+    const options = {
+      title: 'Delete lab result',
+      okLabel: 'Delete',
```

Comment:
can you add `okVariant: 'warning'` so the delete button is red?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/a79a65940f2a0383f7f4d8b28330d2b7e1db30ae

Reply:
my bad this should be 'danger' (so the button is red)

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/files#diff-8147b18fdde2f3d544f4ab613568fc11952197eb2907e5d2a55ae37756aea1a2R22